lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -133,7 +133,7 @@ def join_bulk_responses(bulk_responses)
     end
 
     def bulk_send(body_stream)
-      params = upload_compression ?  {:headers => {"Content-Encoding" => "gzip"}} : {}
+      params = http_compression ?  {:headers => {"Accept-Encoding" => "gzip,deflate", "Content-Encoding" => "gzip"}} : {}
```

Comment:
I don't think it's good for the code to add the Accept-Encoding header in here. In case Manticore drops or adds new algorithms to the set of supported ones we will be sending the wrong headers. Manticore automatically sets the header if the compression is turned on (there's an argument 'compression' which takes a bool and is true by default).